### ISSUE TITLE
Pagination next & last buttons disabled

### ISSUE THREAD

**Issue by Janjuks** [angular-ui/bootstrap#5765]
First of all, I'm sorry, but I cannot reproduce this issue in plunker.
It also doesn't happen always. 
Take a look at the image:
![Image of Yaktocat](http://i.imgur.com/vix67Bs.png)

You cannot see it from screenshot, but the `next` & `last` buttons have disabled class. If I click on one of them, they still work properly.
Directives `total-items` attribute is the same scope variable which is shown below as `117`
There is about 50% chance that this will happen on page refresh.

Logic is as simple as in this plunker :http://plnkr.co/edit/vzp4AWwLmRtKnefMbqKp?p=preview
I have empty object at the beginning and after fetching data from db values are assigned.

Buttons remain disabled even navigation around pages. Only exception is when going to last page in any way and then going back some. Then they appear correctly.

Any help will be appreciated.

Version: `1.1.2`
Angular: `1.4.9`

**Comment by wesleycho** [comment]
Can't really help without a reproduction, we're only guessing at the solution and it's a huge time sink. We have unit tests covering these scenarios though, so I'm pretty certain there is something else going on here.

**Comment by Janjuks** [comment]
I only now noticed that I forgot to change library versions in plunker to the ones I have specified. Which led me to confusion, because - in product we are still using `<pagination>` and I don't see any deprication messages and when I change it to `<uib-pagination>` then it doesn't work, but it's the opposite on plunker. I changed the libraries versions and `<uib-pagination>` to `<pagination>` and it doesn't work. In both cases there is no content generated.
